quipucords/api/management/commands/regenerate_missing_report_cache_files.py

```diff
@@ -71,16 +74,10 @@ def generate_fingerprints(
         """Generate cached fingerprints for the given deployments report IDs."""
         success_ids = []
         failure_ids = []
-        deployments_report_ids = sorted(list(deployments_report_ids))
-        for _id in deployments_report_ids:
-            deployments_report = DeploymentsReport.objects.get(pk=_id)
-            self.write_info(f"Rerunning fingerprint task for {deployments_report}")
-            try:
-                deployments_report._rerun_latest_fingerprint(wait=True)
-                success_ids.append(_id)
-            except Exception as e:  # noqa: BLE001
-                self.stdout.write(self.style.ERROR(str(e)))
-                failure_ids.append(_id)
+        for _id in sorted(list(deployments_report_ids)):
```

Comment:
```suggestion
        for _id in sorted(deployments_report_ids):
```

Calling `sorted()` on a set already returns a list, no need for `list()` call.
(But not like this call is expensive)

Same applies to line 88.

Reply:
Good catch! Simplified in latest push.